library/src/skins/default/src/sass/modules/tool/calendar/_calendarSynoptic.scss

```diff
@@ -7,6 +7,11 @@
 		text-align: center;
 	}
 
+	.tableContainer {
```

Comment:
I would not create and use a new class here. I would try the bootstrap classes for margins. So, mx-2. These classes are defined in bootstrap and can be centrally modified to specify different sizes. So, if we use the bootstrap classes , we get more consistent spacing across more tools in Sakai, and that spacing can be changed in one place in our sass build.

Reply:
Yes, that's a good point, thank your for the suggestion!